src/deploy.sh

```diff
@@ -69,6 +71,9 @@ function kernel_deploy()
   uninstall="${options_values['UNINSTALL']}"
   remote="${remote_parameters['REMOTE']}"
 
+  get_bootloader "$target" 'SILENT'
```

Comment:
Call `learn_device` since we can profit from other data during the deploy; I recommend you to add this `get_bootloader` under `learn_device`.